src/reanimated2/layoutReanimation/web/componentUtils.ts

```diff
@@ -88,6 +90,27 @@ function getReversedFromConfig(config: CustomConfig) {
   return !!config.reversed;
 }
 
+export function extractTransformFromStyle(style: StyleProps) {
+  if (!style) {
+    return;
+  }
+
+  if (typeof style.transform === 'string') {
+    throw new Error('[Reanimated] String transform is currently unsupported.');
+  }
+
+  if (!Array.isArray(style)) {
+    return style.transform;
+  }
+
+  // Only last transform should be considered
```

Comment:
what in situation like that:
```js
style={[{ transform: [{ rotation: 90deg }] }, { transform: [{ scale: 1.5 }] }]}
```

We can try to merge those transforms, or at least we should show a warning to the user that we consider only the last transform from style.

Reply:
If you do something like this, only last transform will be applied to element. I've thought about merging them, but it would be inconsistent with default behavior without reanimated.